pkg/services/sqlstore/migrations/ualert/ualert.go

```diff
@@ -118,6 +118,72 @@ func RerunDashAlertMigration(mg *migrator.Migrator) {
 	}
 }
 
+func AddDashboardUIDPanelIDMigrations(mg *migrator.Migrator) {
+	mg.AddMigration("add dashboard_uid column to alert_rule", migrator.NewAddColumnMigration(
+		migrator.Table{Name: "alert_rule"},
+		&migrator.Column{
+			Name:     "dashboard_uid",
+			Type:     "string",
```

Comment:
```suggestion
			Type: migrator.DB_NVarchar,
			 Length: 40,
```

I think this is the reason that this fails in MySQL and PostgreSQL
```suggestion
			Type:     "string",
```